app/src/main/res/layout/item_join_domain.xml

```diff
@@ -26,110 +26,118 @@
             >
             <LinearLayout
                 android:layout_width="match_parent"
-                android:layout_height="wrap_content"
+                android:layout_height="match_parent"
                 android:layout_margin="@dimen/dp_5"
+                android:background="@drawable/border_comment"
                 android:orientation="vertical"
                 tools:ignore="UselessParent"
                 >
                 <LinearLayout
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
+                    android:background="@drawable/border_comment"
                     android:orientation="horizontal"
+                    android:weightSum="7"
                     tools:ignore="UseCompoundDrawables"
                     >
                     <ImageView
-                        android:layout_width="@dimen/dp_40"
+                        android:layout_width="0dp"
                         android:layout_height="@dimen/dp_60"
+                        android:layout_weight="1"
                         android:contentDescription="@string/domain_location"
                         app:srcCompat="@drawable/ic_location"
                         tools:ignore="ContentDescription"
                         />
                     <TextView
-                        android:layout_width="match_parent"
+                        android:layout_width="0dp"
                         android:layout_height="wrap_content"
                         android:layout_gravity="center"
                         android:layout_marginLeft="@dimen/dp_8"
                         android:layout_marginStart="@dimen/dp_8"
+                        android:layout_weight="4"
+                        android:background="@drawable/border_comment"
                         android:text="@{viewModel.domainName}"
                         android:textColor="@color/color_blue"
                         android:textSize="@dimen/sp_18"
                         android:textStyle="bold"
                         />
+                    <TextView
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="center"
+                        android:layout_marginLeft="@dimen/dp_4"
+                        android:layout_marginStart="@dimen/dp_4"
+                        android:layout_weight="2"
+                        android:gravity="center"
+                        android:padding="@dimen/dp_2"
+                        android:paddingStart="@dimen/dp_10"
```

Comment:
WARNING: [RtlSymmetry] When you define `paddingStart` you should probably also define `paddingEnd` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.